theme_crowdfunding/static/src/style/custom.scss

```diff
@@ -41,6 +41,8 @@ $font-size-sm: $font-size-base * 0.75;
 $font-family-sans-serif: "Montserrat", sans-serif;
 $body-color: #333;
 
+// TODO: Don't apply css globally
```

Comment:
What does it mean? Did you enter a task in Jira for that? Otherwise we risk to forget it.

Reply:
You're right, I will add a Jira task.